scenarios/community/vmware/vmware.yml

```diff
@@ -0,0 +1,191 @@
+vmware:
+  connection:
+  - vmware_tools.py
+  doc_fragments:
+  - VmwareRestModule.py
+  - VmwareRestModule_filters.py
+  - VmwareRestModule_full.py
+  - VmwareRestModule_state.py
+  - vmware.py
+  - vmware_rest_client.py
+  httpapi:
+  - vmware.py
+  inventory:
+  - vmware_vm_inventory.py
+  module_utils:
+  - vmware.py
+  - vmware_httpapi/VmwareRestModule.py
+  - vmware_httpapi/__init__.py
+  - vmware_rest_client.py
+  - vmware_spbm.py
+  modules:
+  - cloud/vmware/__init__.py
```

Comment:
you could glob here, to reduce files if everything in folder is migrated

Reply:
I did it, but this also create a delta with the original configuration.